packages/cli/src/util/paths.ts

```diff
@@ -31,9 +32,35 @@ export function createExamplePath(
 ): E.Either<Error, string> {
   return DoEither.bind('pathData', generateTemplateAndValuesForPathParams(operation))
     .bindL('queryData', ({ pathData }) => generateTemplateAndValuesForQueryParams(pathData.template, operation))
-    .return(({ pathData, queryData }) =>
-      URI.expand(queryData.template, transformValues({ ...pathData.values, ...queryData.values }))
-    );
+    .return(({ pathData, queryData }) => {
+      // replace "-" in template path and query params
+      const cleanedTemplate = queryData.template.replace(/(?<=\{.*)(?=.*\})(-)/g, '');
+      const realValues = transformValues({ ...pathData.values, ...queryData.values });
+      const cleanedValues = {};
+      for (const realValue in realValues) {
+        const cleanedValue = realValue.replace(/-/g, '');
+        cleanedValues[cleanedValue] = realValues[realValue];
+      }
+
+      const cleanedExpandedPath = URI.expand(cleanedTemplate, cleanedValues);
+      const uri = new URIJS(cleanedExpandedPath).escapeQuerySpace(false);
```

Comment:
Why is the `escapeQuerySpace` required to be set to false?

Reply:
Otherwise urijs turns `%20` to `+`. Some of the tests were expecting `%20` instead of `+` in query params, so this kept the output the same as before.

Reply:
see https://github.com/stoplightio/prism/pull/1189#discussion_r428742146